### PR TITLE
Dread: Add some tricks and remove Hanubia block nodes

### PR DESCRIPTION
**Pull request by XenoWars**
- Added: Spin Boost Movement (Intermediate) and Speedbooster Conservation (Beginner) for getting up Hanubia - Central Unit without Space Jump or Infinite Bomb Jump.
- Added: Spin Boost method to climb Hanubia - Escape Room 3.
- Added: Morph Ball Single-Wall Walljumps to get to the Nav Station in Itorash - Transport to Hanubia.
- Changed: Increased difficulty of Flash Shift Walljump to reach the Raven Beak elevator from Intermediate to Advanced.
- Removed all block nodes in Hanubia and the one in Itorash
- Started initial preparation for logic accounting for the ADC and clips for finishing without Hyper Beam

### CODE REVIEW

File: randovania/games/dread/json_data/Hanubia.txt [hunk 1]
```diff
@@ -111,94 +109,38 @@ Extra - asset_id: collision_camera_002
...
-  * Configurable Node
-  * Extra - actor_name: breakabletilegroup_039
-  * Extra - actor_def: actordef:actors/logic/breakabletilegroup/charclasses/breakabletilegroup.bmsad
-  * Extra - tile_types: ('BOMB',)
   > Door to Transport to Ferenia
+      All of the following:
+          Screw Attack
+          Any of the following:
+              Lay Bomb
+              Power Bombs ≥ 2 and Lay Power Bomb
```
Comment (dyceron) [inline randovania/games/dread/json_data/Hanubia.txt:120]: Only need 1 PB

File: randovania/games/dread/json_data/Hanubia.txt [hunk 2]
```diff
@@ -111,94 +109,38 @@ Extra - asset_id: collision_camera_002
...
-  * Extra - actor_def: actordef:actors/logic/breakabletilegroup/charclasses/breakabletilegroup.bmsad
-  * Extra - tile_types: ('BOMB',)
-  > Tile Group (BOMB) 4
-      Morph Ball
-  > Missile Tank Alcove
-      Morph Ball
+  > Tunnel to Speedboost Puzzle Room
+      Any of the following:
+          Lay Bomb
+          Power Bombs ≥ 2 and Lay Power Bomb
```
Comment (dyceron) [inline randovania/games/dread/json_data/Hanubia.txt:126]: Same as above

File: randovania/games/dread/json_data/Hanubia.txt [hunk 3]
```diff
@@ -595,8 +519,10 @@ Extra - asset_id: collision_camera_009
   * Pickup 135; Major Location? False
   * Extra - actor_name: item_powerbombtank_000
   * Extra - actor_def: actordef:actors/items/item_powerbombtank/charclasses/item_powerbombtank.bmsad
-  > Tile Group (BOMB) 2
-      Trivial
+  > Door from Tank Room
+      Any of the following:
+          Lay Bomb
+          Power Bombs ≥ 2 and Lay Power Bomb
```
Comment (dyceron) [inline randovania/games/dread/json_data/Hanubia.txt:525]: Same as above

File: randovania/games/dread/json_data/Hanubia.txt [hunk 4]
```diff
@@ -973,24 +860,28 @@ Extra - asset_id: collision_camera_015
...
 
 > Dock to Escape Room 2; Heals? False
   * Layers: default
   * Open Passage to Escape Room 2/Dock to Total Recharge Station East
-  > Tile Group (POWERBOMB) 3
-      Trivial
+  > Dock to EMMI Zone Exit East (Lower)
+      Any of the following:
+          After Escape Sequence
+          Power Bombs ≥ 4 and Lay Power Bomb
```
Comment (dyceron) [inline randovania/games/dread/json_data/Hanubia.txt:884]: Why 4? There's only 2 blocks before the lower door, 1 of which is breakable from the previous escape room.

**Review by dyceron — CHANGES_REQUESTED** [review]

**Comment by XenoWars** [comment]
All the cases of PBs being more than 1 is to ensure you do not get stuck in that area. Like in Shortcut to Ferenia, you can get in the alcove with one power bomb, but you'll need another to get out.

**Comment by dyceron** [comment]
> All the cases of PBs being more than 1 is to ensure you do not get stuck in that area. Like in Shortcut to Ferenia, you can get in the alcove with one power bomb, but you'll need another to get out.

I hadn't considered that. PR seems fine then.

**Review by dyceron — APPROVED** [review]

**Pull request merged**